test/influxdb_v2/deleter/predicate_test.exs

```diff
@@ -0,0 +1,122 @@
+defmodule Instream.InfluxDBv2.Deleter.PredicateTest do
+  use ExUnit.Case, async: true
+
+  @moduletag :"influxdb_include_2.x"
+
+  alias Instream.TestHelpers.TestConnection
+
+  defmodule EmptyTagSeries do
+    use Instream.Series
+
+    series do
+      measurement "empty_predicate_tags"
+
+      tag :filled
+      tag :defaulting, default: "default_value"
+      tag :empty
+
+      field :value
+    end
+  end
+
+  describe "testing" do
```

Comment:
Well, I would say every test ist testing something so we can drop that `:describe` :sweat_smile:

Reply:
Ah the placeholder. It was intended to be `describe "delete/2" do`
But you are right, we should drop `:describe`